packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -45,26 +45,31 @@ nls.config({
 })();
 const localize: nls.LocalizeFunc = nls.loadMessageBundle();
 
+let typeEnum: zowe.CreateDataSetTypeEnum;
+// Make a nice new mutable array for the DS properties
+let newDSProperties;
+
 /**
  * Localized strings that are used multiple times within the file
  */
 const localizedStrings = {
-    dsBinary: localize("dsActions.dataSetBinary", "Data Set Binary"),
-    dsC: localize("dsActions.dataSetC", "Data Set C"),
-    dsClassic: localize("dsActions.dataSetClassic", "Data Set Classic"),
-    dsPartitioned: localize("dsActions.dataSetPartitioned", "Data Set Partitioned"),
-    dsSequential: localize("dsActions.dataSetSequential", "Data Set Sequential"),
-    dsActionCancelled: localize("dsActions.cancelled", "Operation Cancelled"),
+    dsBinary: localize("createFile.dataSetBinary", "Partitioned Data Set: Binary"),
+    dsC: localize("createFile.dataSetC", "Partitioned Data Set: C"),
+    dsClassic: localize("createFile.dataSetClassic", "Partitioned Data Set: Classic"),
+    dsPartitioned: localize("createFile.dataSetPartitioned", "Partitioned Data Set: Default"),
+    dsSequential: localize("createFile.dataSetSequential", "Sequential Data Set"),
+    opCancelled: localize("dsActions.cancelled", "Operation Cancelled"),
     copyingFiles: localize("dsActions.copy.inProgress", "Copying File(s)"),
     profileInvalid: localize("dsActions.profileInvalid", "Profile is invalid, check connection details."),
+    allocString: localize("dsActions.allocate", "Allocate Data Set"),
+    editString: localize("dsActions.editAttributes", "Edit Attributes"),
```

Comment:
Would it help to add icons for these actions so that they stand out from the other options in the quick pick? It would also match our current design choice for the dataset and USS quick picks.

We can also leverage [codicons](https://microsoft.github.io/vscode-codicons/dist/codicon.html) here as they are [supported](https://code.visualstudio.com/api/references/icons-in-labels#icons-in-labels) in both VS Code and Theia. We are currently using those for some of the quick picks.

Reply:
These are just the localized strings I have the codicon values at the location of dropdown list creation.  Had this one already in place.
![Screen Shot 2023-05-19 at 10 44 28 AM](https://github.com/zowe/vscode-extension-for-zowe/assets/49491949/380367dd-e568-44cb-9514-44a8eaba6394)

Adding this when pushing updates for other comments
![Screen Shot 2023-05-19 at 10 49 41 AM](https://github.com/zowe/vscode-extension-for-zowe/assets/49491949/d8d41d42-79a7-4dfd-ae40-2749584e3c81)